### DISCUSSION TITLE
CMAKE PROBLEMA

### DISCUSSION THREAD

**Discussion by andromeda2075** [cpp-review-dune/introductory-review#17] [Q&A] [answered]
Consulta acerca de cmake: 
https://asciinema.org/a/470257

**Comment by carlosal1015** [comment, marked as answer]
Dado que la función [`add_executable`](https://cmake.org/cmake/help/v3.23/command/add_executable.html) necesita como mínimos 2 parámetros (_separados por espacios_)
```cmake
add_executable(<name> [WIN32] [MACOSX_BUNDLE]
               [EXCLUDE_FROM_ALL]
               [source1] [source2 ...])
```
y tenemos un único parámetro en la función `add_executable`, porque las comillas dobles considera "array1 arrays.cc" como solo un parámetro, este falla.
```cmake
cmake_minimum_required(VERSION 3.13)
project("Ejemplo")
add_executable("array1 arrays.cc")
add_executable("array2 arrays2.cc")
```
De esta forma se podría solucionar.
```cmake
cmake_minimum_required(VERSION 3.13)
project("Ejemplo")
add_executable("array1" "arrays.cc")
add_executable("array2" "arrays2.cc")
```
